Bottom: 7f698e53594e2646b13306cf658592a66772ed7d
Top:    3ae8ca2920679b0534903df83399a5a4275a380d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 09:25:45 -0500

Refresh of tmp-wiki-database

---
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 868e5917..273f43c4 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -122,10 +122,10 @@
       --user={{ wiki_db_user | quote }}
       --password={{ wiki_db_password | quote }}
       '{{ wiki_database_name }}'
-  register: container_status
+  register: db_status
   retries: 20
   delay: 3
-  until: "container_status.rc == 0"
+  until: "db_status.rc == 0"
 
 # *****************************************************************************
 # Start the data container running
